scripts/run_install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
dynaflow-launcher.sh would be a better name

Reply:
OK